src/form-elements/FormElementABN.tsx

```diff
@@ -246,6 +246,7 @@ function FormElementABN({
               }}
               onFocus={setIsFocused}
               aria-describedby={ariaDescribedby}
+              autoComplete={element.autocompleteAttributes?.join(' ')}
```

Comment:
Let's do this once in OneBlinkFormElements and it will either a string or undefined